### ISSUE TITLE
benchmark result 

### ISSUE THREAD

**Issue by Cherryjingyao** [mees/calvin#73]
I found there is a benchmark: http://calvin.cs.uni-freiburg.de/
Is this a public open benchmark?
But I can not find the  submission  place .Where can we submit  the evaluation results?

**Comment by lukashermann** [comment]
Yes, you can run the CALVIN evaluation as described in the README [here](https://github.com/mees/calvin?tab=readme-ov-file#muscle-evaluation-the-calvin-challenge) and if you get a performance that beats our baselines (incl. HULC), then contact [Oier](https://www.oiermees.com/) to add your model to the leaderboard.
